modules/osdk-updating-v125-to-v128.adoc

```diff
@@ -41,10 +41,10 @@ The following procedure updates an existing {type}-based Operator project for co
 .Procedure
 
 ifdef::helm,hybrid,java[]
-* Find the `ose-kube-rbac-proxy` pull spec in the following files, and update the image tag to `v4.13`:
+* Find the `ose-kube-rbac-proxy` pull spec in the following files, and update the image tag to `v4.14`:
```

Comment:
Same comment as prior.

Reply:
Same comment as above - this file has both 4.13 and 4.14, so we couldn't replace with attributes.